$OpenBSD: patch-tests_CMakeLists_txt,v 1.2 2019/08/13 18:10:11 phessler Exp $

test-bit_string_*.cpp depends on signed char, failing on aarch64

Index: tests/CMakeLists.txt
--- tests/CMakeLists.txt.orig
+++ tests/CMakeLists.txt
@@ -3,7 +3,6 @@ include_directories(
 	${Boost_INCLUDE_DIRS}
 	${CMAKE_CURRENT_SOURCE_DIR}/../src
 )
-add_subdirectory(${GTEST_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/gtest_build)
 
 set(tests_SOURCES
 	main.cpp
@@ -15,8 +14,6 @@ set(tests_SOURCES
 	test_integral_image.cpp
 	test_lloyds.cpp
 	test_audio_processor.cpp
-	test_bit_string_reader.cpp
-	test_bit_string_writer.cpp
 	test_chromaprint.cpp
 	test_chroma.cpp
 	test_chroma_filter.cpp
